oauth2.go

```diff
@@ -105,6 +105,25 @@ func (s *Session) ApplicationDelete(appID string) (err error) {
 	return
 }
 
+// Asset struct stores values for an asset of an application
+type Asset struct {
+	Type int    `json:"type"`
+	ID   string `json:"id"`
+	Name string `json:"name"`
+}
+
+// ApplicationAssets returns an application's assets
+func (s *Session) ApplicationAssets(appID string) (ass []*Asset, err error) {
```

Comment:
Why is Asset a pointer here? On each index access there's gonna be a double pointer dereference, and since it's a slice the contents won't get copied anyway.
```suggestion
func (s *Session) ApplicationAssets(appID string) (ass []Asset, err error) {
```

Reply:
Slice of pointers is consistent with the rest of the library.